djangocms_navigation/admin.py

```diff
@@ -241,8 +273,15 @@ def move_node(self, request, menu_content_id):
         # Disallow moving of a node on anything other than a draft version
         if self._versioning_enabled:
             menu_content = get_object_or_404(
-                MenuContent._base_manager, id=menu_content_id
+                self.MenuModel._base_manager, id=menu_content_id
             )
+            request.menu_content_id = menu_content_id
+            change_perm = self.has_change_permission(request, menu_content)
+            if not change_perm:
+                msg = 'The item is currently locked or you don\'t have permission to change it'
```

Comment:
The should use the get text _("")

I think the message should also be different for version locking and when version locking is not installed e.g. 

Version locking enabled: _("The item is currently locked or you don\'t have permission to change it")
Version locking disabled: _("You don\'t have permission to change this item")